README.md

```diff
@@ -89,6 +89,7 @@ Read some additional details about 'moderator mode' on the [wiki page here](http
 ## Installation
 
 If using the python script version (not the exe), there is a requirements.txt with necessary modules. Created with Python 3.9.7
+The Windows .EXE version requires Windows 8 or above [(Why?)](https://pyinstaller.readthedocs.io/en/stable/requirements.html#windows). This can be fixed by installing Microsoft's "Universal C Runtime" for your system [here](https://support.microsoft.com/en-us/topic/update-for-universal-c-runtime-in-windows-c0514201-7fe6-95a3-b0a5-287930f3560c).
```

Comment:
```suggestion
The Windows .EXE version may not run on systems older than Windows 8 [(Why?)](https://pyinstaller.readthedocs.io/en/stable/requirements.html#windows). This can be fixed by installing Microsoft's "Universal C Runtime" for your system [here](https://support.microsoft.com/en-us/topic/update-for-universal-c-runtime-in-windows-c0514201-7fe6-95a3-b0a5-287930f3560c).
```
Just to match the "This can be fixed"

Reply:
Yeah, also for some computers it may run?

Reply:
I think so, if they have the update installed

Reply:
Yeah, so it doesnt make sense to say "requires Windows 8 or above" anyway.